Program.cs

```diff
@@ -2,39 +2,155 @@
 using Google.Apis.Drive.v3;
 using Google.Apis.Services;
 using Google.Apis.Util.Store;
+using System.Diagnostics;
 
 namespace FileUploadToDrive
 {
     internal class Program
     {
-        internal static async Task Main(string[] args)
+        internal static IList<string> files = new List<string>();
+        internal const string jsonPath = "F:\\Prithvi\\Projects\\FileUploadToDriveFiles\\Credentials.json";
```

Comment:
Suggestion: Remove folder paths and take them as inputs.

Reply:
I have made the changes. Can you please review?